packages/styling/src/classNames/ColorClassNames.ts

```diff
@@ -6,297 +6,297 @@ import { DefaultPalette } from '../styles/DefaultPalette';
+  redBackgroundHover: string;
+  redBorder: string;
+  redBorderHover: string;
+  magentaDark: string;
+  magentaDarkHover: string;
+  magentaDarkBackground: string;
+  magentaDarkBackgroundHover: string;
+  magentaDarkBorder: string;
+  magentaDarkBorderHover: string;
+  magenta: string;
+  magentaHover: string;
+  magentaBackground: string;
+  magentaBackgroundHover: string;
+  magentaBorder: string;
+  magentaBorderHover: string;
+  magentaLight: string;
+  magentaLightHover: string;
+  magentaLightBackground: string;
+  magentaLightBackgroundHover: string;
+  magentaLightBorder: string;
+  magentaLightBorderHover: string;
+  purpleDark: string;
+  purpleDarkHover: string;
+  purpleDarkBackground: string;
+  purpleDarkBackgroundHover: string;
+  purpleDarkBorder: string;
+  purpleDarkBorderHover: string;
+  purple: string;
+  purpleHover: string;
+  purpleBackground: string;
+  purpleBackgroundHover: string;
+  purpleBorder: string;
+  purpleBorderHover: string;
+  purpleLight: string;
+  purpleLightHover: string;
+  purpleLightBackground: string;
+  purpleLightBackgroundHover: string;
+  purpleLightBorder: string;
+  purpleLightBorderHover: string;
+  blueDark: string;
+  blueDarkHover: string;
+  blueDarkBackground: string;
+  blueDarkBackgroundHover: string;
+  blueDarkBorder: string;
+  blueDarkBorderHover: string;
+  blueMid: string;
+  blueMidHover: string;
+  blueMidBackground: string;
+  blueMidBackgroundHover: string;
+  blueMidBorder: string;
+  blueMidBorderHover: string;
+  blue: string;
+  blueHover: string;
+  blueBackground: string;
+  blueBackgroundHover: string;
+  blueBorder: string;
+  blueBorderHover: string;
+  blueLight: string;
+  blueLightHover: string;
+  blueLightBackground: string;
+  blueLightBackgroundHover: string;
+  blueLightBorder: string;
+  blueLightBorderHover: string;
+  tealDark: string;
+  tealDarkHover: string;
+  tealDarkBackground: string;
+  tealDarkBackgroundHover: string;
+  tealDarkBorder: string;
+  tealDarkBorderHover: string;
+  teal: string;
+  tealHover: string;
+  tealBackground: string;
+  tealBackgroundHover: string;
+  tealBorder: string;
+  tealBorderHover: string;
+  tealLight: string;
+  tealLightHover: string;
+  tealLightBackground: string;
+  tealLightBackgroundHover: string;
+  tealLightBorder: string;
+  tealLightBorderHover: string;
+  greenDark: string;
+  greenDarkHover: string;
+  greenDarkBackground: string;
+  greenDarkBackgroundHover: string;
+  greenDarkBorder: string;
+  greenDarkBorderHover: string;
+  green: string;
+  greenHover: string;
+  greenBackground: string;
+  greenBackgroundHover: string;
+  greenBorder: string;
+  greenBorderHover: string;
+  greenLight: string;
+  greenLightHover: string;
+  greenLightBackground: string;
+  greenLightBackgroundHover: string;
+  greenLightBorder: string;
+  greenLightBorderHover: string;
 }
 
-export const ColorClassNames: IColorClassNames = {};
+export const ColorClassNames: Partial<IColorClassNames> = {};
```

Comment:
Shouldn't this be:

```tsx
export const ColorClassNames: IColorClassNames = {} as IColorClassNames;
```

Then in your code, you don't need to treat it as an non-optional?